bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -49,17 +47,19 @@
     // information about the binding config of the items)
     private BticinoBinding m_bticino_binding;
     // Hostname or Host IP (of the MH200)
-    private String m_host = "";
+    private String m_host = "192.168.3.35";
```

Comment:
This should go back to the original empty string value.

Reply:
done